src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor

```diff
@@ -19,7 +19,7 @@
     }
     else if (Resource is ExecutableViewModel executableViewModel)
     {
-        var title = $"PID: {executableViewModel.ProcessId}";
+        var title = $"Process Id: {executableViewModel.ProcessId}";
```

Comment:
```suggestion
        var title = $"Process ID: {executableViewModel.ProcessId}";
```

Reply:
but container uses Container Id in tool tip but ID in tool tip copy. Inconsistency will trigger some OCD at some point.

Reply:
Hence my other comment above that we have a few places we need to clean up 😊